Assets/MRTK/SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -213,7 +211,7 @@ private bool FindClosestTouchableForLayerMask(LayerMask layerMask, out BaseNearI
             {
                 closest = null;
                 closestDistance = float.PositiveInfinity;
-                closestNormal = Vector3.zero;
+                closestNormal = Rotation * Vector3.forward;
```

Comment:
With this change, it looks like we're now doing this quaternion math once for each layer mask in `PrioritizedLayerMasksOverride` instead of just once per `OnPreSceneQuery()` call, which seems like it could be worse on perf. Should this calculation be moved to something like the `else` case for `if (newClosestTouchable != null)` in `OnPreSceneQuery()`? That way we're only doing the multiplication _at most_ once per frame, and that should solidify the contract a bit that we can expect the value to be unchanged (and thus reusable!) when `TryGetNormalToNearestSurface` is called.

Reply:
That's an excellent point. I've moved the quarternion math to `OnPreSceneQuery()` like you suggest.